scripts/exact-candidate-qualification.sh

```
@@ -0,0 +1,213 @@
+
+echo "exact-candidate-qualification: running the supported journey from the isolated binary"
+CARGO_ALLOW_BIN="$ROOT/$INSTALLED_BIN" \
+WORK_DIR="${OUTPUT_DIR}/journey" \
+bash scripts/exact-candidate-install-journey.sh
+
+journey_receipt="${OUTPUT_DIR}/journey/source-candidate-smoke.receipt.json"
+[ -f "$journey_receipt" ] || fail "journey did not emit ${journey_receipt}"
+
+echo "exact-candidate-qualification: assembling the typed receipt"
+commit="$(git rev-parse HEAD)"
+tree="$(git rev-parse HEAD^{tree})"
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)